src/qtism/data/content/xhtml/ObjectElement.php

```diff
@@ -158,79 +164,62 @@ public function getType()
 
     /**
      * Set the width of the object.
+     * A null value means that no width is set.
      *
-     * A negative value describes that no width is provided.
-     *
-     * @param int $width A width.
-     * @throws InvalidArgumentException
+     * @param mixed $width A width.
+     * @throws InvalidArgumentException when $width is not valid.
      */
-    public function setWidth($width)
+    public function setWidth($width): void
     {
-        if (is_int($width)) {
-            $this->width = $width;
-        } else {
-            $msg = "The 'width' argument must be an integer, '" . gettype($width) . "' given.";
-            throw new InvalidArgumentException($msg);
-        }
+        $this->width = Format::sanitizeXhtmlLength($width, 'width');
     }
 
     /**
      * Get the width of the object.
-     *
-     * A negative value describes that no width is provided.
-     *
-     * @return int A width.
+     * Null means that no width is set.
```

Comment:
I think it is unnecessary or confusing comment, now return type says everything, and cause the class contains "has" methods it would be correct to say that their responsibility to answer whether width been set.
(Same for similar getters)

Reply:
Thanks for remark. Done!